worlds/messenger/docs/setup_en.md

```diff
@@ -18,17 +18,17 @@ Read changes to the base game on the [Game Info Page](/games/The%20Messenger/inf
 3. Click on "The Messenger"
 4. Follow the prompts
 
+These steps can be followed in the future to launch the game and check for mod updates.
+
 ### Manual Installation
 
 1. Download and install Courier Mod Loader using the instructions on the release page
    * [Latest release is currently 0.7.1](https://github.com/Brokemia/Courier/releases)
 2. Download and install the randomizer mod
    1. Download the latest TheMessengerRandomizerAP.zip from
-      [The Messenger Randomizer Mod AP releases page](https://github.com/alwaysintreble/TheMessengerRandomizerModAP/releases)
+[The Messenger Randomizer Mod AP releases page](https://github.com/alwaysintreble/TheMessengerRandomizerModAP/releases)
    2. Extract the zip file to `TheMessenger/Mods/` of your game's install location
-      * You cannot have both the non-AP randomizer and the AP randomizer installed at the same time. The AP randomizer
-        is backwards compatible, so the non-AP mod can be safely removed, and you can still play seeds generated from the
-        non-AP randomizer.
+      * You cannot have both the non-AP randomizer and the AP randomizer installed at the same time
```

Comment:
Just to make sure, part about backwards compatibility was removed voluntary? You do not want to provide further information about how one can ping-pong between the two?

Reply:
In this instance, it seems to be expected that the player is here for Archipelago, possibly coming from the standalone. However, AP supports a single-world randomizer so seems redundant to talk about the standalone unless there was a difference in features.

Reply:
This version of the randomizer functionally replaces the old one at this point. There was still a bit of utility in playing the standalone back when this was written, since with the AP randomizer you always had to connect to a server to play, but this is no longer the case.